src/utils/utils.ts

```diff
@@ -62,7 +63,7 @@ export const loadNeverminedConfigContract = (config: ConfigEntry): Contract => {
 }
 
 export const loadNFT1155Contract = (config: ConfigEntry, address: string | undefined): Contract => {
```

Comment:
can't we just use `nevermined.contracts.loadNft1155` instead of this?

Reply:
code removed as suggested